Doc/library/pathlib.rst

```diff
@@ -933,23 +933,26 @@ call fails (for example because the path doesn't exist).
 
 .. method:: Path.rename(target)
 
-   Rename this file or directory to the given *target*.  On Unix, if
-   *target* exists and is a file, it will be replaced silently if the user
-   has permission.  *target* can be either a string or another path object::
+   Rename this file or directory to the given *target*, and return a new Path
```

Comment:
Please add a `versionchanged` directive to note the change in return value behavior.

Reply:
Done, thanks tirkarthi.